fvm/src/machine/engine.rs

```diff
@@ -208,6 +208,33 @@ struct Cache<K> {
 }
 
 impl Engine {
+    /// Attempts to load Actor Code CID from the blockstore and
+    /// instantiate a wasmtime Module from the WASM bytecode if the
+    /// given code CID is present, otherwise returns None.
+    ///
+    /// Implicitly caches instantiated modules.
+    pub fn load_code_cid<BS: Blockstore>(
```

Comment:
`install_code`? We don't actually need to return it.

Reply:
SGTM. I'll move this part to a different patch and we can address there.